extensions-contrib/opencensus-extensions/src/test/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParserTest.java

```diff
@@ -124,6 +125,32 @@ public void testGaugeParse() throws Exception
     Assert.assertEquals(2000, row.getMetric("value").doubleValue(), 0.0);
   }
 
+  @Test
+  public void testIntGaugeParse() throws Exception
+  {
+    //configure parser with desc file
+    OpenCensusProtobufInputRowParser parser = new OpenCensusProtobufInputRowParser(parseSpec, null, null, "");
+
+    DateTime dateTime = new DateTime(2019, 07, 12, 9, 30, ISOChronology.getInstanceUTC());
+
+    Timestamp timestamp = Timestamp.newBuilder().setSeconds(dateTime.getMillis() / 1000)
+        .setNanos((int) ((dateTime.getMillis() % 1000) * 1000000)).build();
+
+    System.out.println(timestamp.getSeconds() * 1000);
+
+    Metric metric = intGaugeMetric(timestamp);
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
+    metric.writeTo(out);
+
+    InputRow row = parser.parseBatch(ByteBuffer.wrap(out.toByteArray())).get(0);
+    Assert.assertEquals(dateTime.getMillis(), row.getTimestampFromEpoch());
+
+    assertDimensionEquals(row, "name", "metric_gauge_int64");
+    assertDimensionEquals(row, "foo_key", "foo_value");
+
+    Assert.assertEquals(1000, row.getMetric("value").intValue(), 0.0);
```

Comment:
for integer comparisons, the delta (last argument to assertEquals) is unnecessary

Reply:
Done.